week08_pomdp/practice_pytorch.ipynb

```diff
@@ -256,19 +255,36 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def evaluate(agent, env, n_games=1):\n",
+    "def evaluate(agent,\n",
+    "             env,\n",
+    "             video_dir: Union[Text, None]=None,\n",
```

Comment:
`Union[Text, None]` is called `Optional[str]` (I'm not sure what `Text` even is).